package/MDAnalysis/coordinates/GRO.py

```diff
@@ -163,23 +140,23 @@ def _read_first_frame(self):
             # Always try, and maybe add them later
             velocities = np.zeros((n_atoms, 3), dtype=np.float32)
 
-            self.ts = ts = self._Timestep(n_atoms,
-                                          **self._ts_kwargs)
-
+            self.ts = ts = base.Timestep(n_atoms, **self._ts_kwargs)
             missed_vel = False
 
             grofile.seek(0)
             for pos, line in enumerate(grofile, start=-2):
                 # 2 header lines, 1 box line at end
                 if pos == n_atoms:
-                    unitcell = np.float32(line.split())
+                    unitcell = list(map(float, line.split()))
```

Comment:
I thought we benchmarked those to use `np.float32`?

Reply:
Yes, @kain88-de is right. We determined that the fastest conversion is letting numpy do everything.

Reply:
yes